pubsublite-spark-sql-streaming/src/main/java/com/google/cloud/pubsublite/spark/PslContinuousInputPartition.java

```diff
@@ -50,8 +51,9 @@ public PslContinuousInputPartition(
 
   @Override
   public InputPartitionReader<InternalRow> createContinuousReader(PartitionOffset offset) {
-    assert SparkPartitionOffset.class.isAssignableFrom(offset.getClass())
-        : "offset is not assignable to SparkPartitionOffset";
+    Preconditions.checkState(
```

Comment:
nit. here and elsewhere: import static com.google.common.base.Preconditions.check<State,Argument>